pkg/kademlia/kademlia.go

```diff
@@ -183,7 +184,7 @@ func (k *Kademlia) Bootstrap(ctx context.Context) (err error) {
 			// The way FetchPeerIdentityUnverified does is is to do a basic ping request, which
 			// we have now done. Let's tell all the transport observers now.
 			// TODO: remove the explicit transport observer notification
-			k.dialer.transport.AlertSuccess(ctx, &pb.Node{
+			transport.AlertSuccess(ctx, &pb.Node{
```

Comment:
Was there a reason dialer isn't alerting itself?

Reply:
Apparently it has something to do with the comment above it, but I don't see why we couldn't just call this inside `FetchPeerIdentityUnverified`. Waiting for @jtolds to have some free time to discuss

Reply:
I added a wrapper to call this from dial package